src/com/flickity.js

```diff
@@ -1,7 +1,14 @@
 /**
  * @author Deux Huit Huit
  *
- *	Flickity component
+ * Flickity component
+ *
+ * @uses jQuery.fn.data()
+ *  The used keys are:
+ *   1. `acc-tabindex` used to store the tabindex
+ *   2. `flickity` used to store all the data-flickity-* options
+ *
+ * @requires https://cdnjs.cloudflare.com/ajax/libs/flickity/2.0.10/flickity.pkgd.min.js"
```

Comment:
Change the version for this one.
https://cdnjs.cloudflare.com/ajax/libs/flickity/2.1.1/flickity.pkgd.min.js

Reply:
@fhamon check b94ecc682d